vita3k/config/include/config/config.h

```diff
@@ -148,6 +148,7 @@ enum ScreenshotFormat {
     code(bool, "display-info-message", false, display_info_message)                                     \
     code(bool, "show-welcome", true, show_welcome)                                                      \
     code(bool, "check-for-updates", true, check_for_updates)                                            \
+    code(bool, "dencrypt-install", true, dencrypt_install)                                              \
```

Comment:
Why true ?
Is not normal situation decrypt selfie in install, you get broken self after